pkg/apis/batch/v1/conversion.go

```diff
@@ -32,15 +32,13 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 			case "metadata.name", "metadata.namespace", "status.successful":
 				return label, value, nil
 			default:
-				return "", "", fmt.Errorf("field label %q not supported for batchv1.Job", label)
+				return "", "", fmt.Errorf("field label %q not supported for Job", label)
 			}
-		},
-	)
+		})
 }
 
 // The following functions don't do anything special, but they need to be added
 // here due to the dependency of v1beta1 and v2alpha1 on v1.
-
```

Comment:
revert empty line, as this comment applies to all subsequent functions and not just the next one.

Reply:
you can remove the mention of `v2alpha1` in the comment now

Reply:
Updated both in the latest push.